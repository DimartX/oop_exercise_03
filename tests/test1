help
read 
0 1
1 2
2 1
1 0
figures
centers
read
0 1
2 3
3 2
1 0
figures
read
1 2
6 7
9 8
areas
figures
sum_area
read    
1.3 4.6
0.1 5.7 
9.123 3.1415
figures
centers
delete 1
figures
delete 5
delete 3
figures
exit

