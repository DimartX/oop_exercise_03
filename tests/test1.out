read, delete, areas, centers, figures, sum_area, help, exit
Figures: 
Square: 
{0.000000 1.000000} 
{1.000000 2.000000} 
{2.000000 1.000000} 
{1.000000 0.000000} 

Centers: 
{1.000000 1.000000} 
Figures: 
Square: 
{0.000000 1.000000} 
{1.000000 2.000000} 
{2.000000 1.000000} 
{1.000000 0.000000} 

Rectangle: 
{0.000000 1.000000} 
{2.000000 3.000000} 
{3.000000 2.000000} 
{1.000000 0.000000} 

Areas: 
2.000000
6.000000
2.500000
Figures: 
Square: 
{0.000000 1.000000} 
{1.000000 2.000000} 
{2.000000 1.000000} 
{1.000000 0.000000} 

Rectangle: 
{0.000000 1.000000} 
{2.000000 3.000000} 
{3.000000 2.000000} 
{1.000000 0.000000} 

Triangle: 
{1.000000 2.000000} 
{6.000000 7.000000} 
{9.000000 8.000000} 

Summary area: 
10.500000
Figures: 
Square: 
{0.000000 1.000000} 
{1.000000 2.000000} 
{2.000000 1.000000} 
{1.000000 0.000000} 

Rectangle: 
{0.000000 1.000000} 
{2.000000 3.000000} 
{3.000000 2.000000} 
{1.000000 0.000000} 

Triangle: 
{1.000000 2.000000} 
{6.000000 7.000000} 
{9.000000 8.000000} 

Triangle: 
{1.300000 4.600000} 
{0.100000 5.700000} 
{9.123000 3.141500} 

Centers: 
{1.000000 1.000000} 
{1.500000 1.500000} 
{5.333333 5.666667} 
{3.507667 4.480500} 
Figures: 
Rectangle: 
{0.000000 1.000000} 
{2.000000 3.000000} 
{3.000000 2.000000} 
{1.000000 0.000000} 

Triangle: 
{1.000000 2.000000} 
{6.000000 7.000000} 
{9.000000 8.000000} 

Triangle: 
{1.300000 4.600000} 
{0.100000 5.700000} 
{9.123000 3.141500} 

There is no such element!
Figures: 
Rectangle: 
{0.000000 1.000000} 
{2.000000 3.000000} 
{3.000000 2.000000} 
{1.000000 0.000000} 

Triangle: 
{1.000000 2.000000} 
{6.000000 7.000000} 
{9.000000 8.000000} 

